ykman/openpgp.py

```diff
@@ -375,6 +387,20 @@ def get_remaining_pin_tries(self):
         data = self._get_data(DO.PW_STATUS)
         return PinRetries(*data[4:7])
 
+    def get_signature_pin_policy(self):
+        data = self._get_data(DO.PW_STATUS)
+
+        if bool(data[0]):
+            return POLICY.ONCE
+        else:
+            return POLICY.ALWAYS
+
+    def set_signature_pin_policy(self, status):
+        """Requires Admin PIN verification."""
+        data = struct.pack(">B", status == POLICY.ONCE)
```

Comment:
`status` should be renamed to `pin_policy`. The equals-check is not needed, you can just use `struct.pack(">B", pin_policy)`.